Multisig/UI/Transactions/TransactionDetails/LoadingTransactionDetailsViewModel.swift

```diff
@@ -73,4 +73,31 @@ class LoadingTransactionDetailsViewModel: ObservableObject {
 
         }
     }
+
+    func sign() {
+        guard let transferTx = transactionDetails as? TransferTransactionViewModel,
+              let transaction = transferTx.transaction else {
+            preconditionFailure("We have a technical problem. You need to restart the app.")            
+        }
+        status = .loading
+        Just(safe.address!)
```

Comment:
Do we need a safe address? Can we take the safe address not from the CoreData but from the transaction itself? That would make this signing functionality more flexible and still not so complicated.

Reply:
I moved getting safe here from class property to a variable. TransactionViewModel does not know about safe